conf.py

```diff
@@ -247,7 +247,10 @@
 # One or more folders containing files to be copied as-is into the output.
 # The format is a dictionary of {source: relative destination}.
 # Default is:
-# FILES_FOLDERS = {'files': ''}
+FILES_FOLDERS = {
+    'files/talks': 'talks',
+    #'files': ''
```

Comment:
The comment above seems to suggest that `'files': ''` should do exactly what you want if you create a `talks` folder within files?

Reply:
uhm, maybe. Let me try :)

Reply:
indeed, it works. See d27bd77, thanks